openshift-knative-operator/Dockerfile

```diff
@@ -1,19 +1,37 @@
-FROM registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.22-openshift-4.17 AS builder
+# DO NOT EDIT! Generated Dockerfile for openshift-knative-operator/cmd/openshift-knative-operator.
+ARG GO_BUILDER=registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.21-openshift-4.17
+ARG GO_RUNTIME=registry.access.redhat.com/ubi8/ubi-minimal
 
-ENV BASE=github.com/openshift-knative/serverless-operator
-WORKDIR ${GOPATH}/src/${BASE}
+FROM $GO_BUILDER as builder
 
 COPY . .
 
-ENV GOFLAGS="-mod=vendor"
-RUN go build -o /tmp/operator ${BASE}/openshift-knative-operator/cmd/operator
+ENV CGO_ENABLED=1
+ENV GOEXPERIMENT=strictfipsruntime
+
+RUN go build -tags strictfipsruntime -o /usr/bin/main ./openshift-knative-operator/cmd/openshift-knative-operator
+
+FROM $GO_RUNTIME
+
+ARG VERSION=
```

Comment:
Ah. It's taken from `project.tag` in the project.yaml. But this element is not there in serverless-operator. 
I think it's OK to leave it empty. It will be overridden by build arguments in Tekton pipeline.